addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -505,3 +505,21 @@ registry.category("web_tour.tours").add("ProductSearchTour", {
             ProductScreen.productIsDisplayed("Test Product 2"),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("ProductCardUoMPrecision", {
+    checkDelay: 50,
+    steps: () =>
+        [
+            Chrome.startPoS(),
+            Dialog.confirm("Open Register"),
+            ProductScreen.clickDisplayedProduct("Configurable Chair", false, "1", "5.10"),
+            ProductConfiguratorPopup.pickRadio("Leather"),
+            Chrome.clickBtn("Add"),
+            inLeftSide([Numpad.click("1"), Numpad.click("."), Numpad.click("9")]),
+            ProductScreen.clickDisplayedProduct("Configurable Chair", false, "1", "10.20"),
+            ProductConfiguratorPopup.pickRadio("wool"),
+            Chrome.clickBtn("Add"),
+            inLeftSide([Numpad.click("2"), Numpad.click("."), Numpad.click("3")]),
```

Comment:
If we skip the mobile version we don't need the `inLeftSide`. But I'm still curious to understand why the behavior is different on mobile for the tour. Maybe @caburj have an idea why the tour on mobile is only working with a bigger delay